mirascope/cli/utils.py

```diff
@@ -223,10 +236,82 @@ def get_prompt_analyzer(file: str) -> PromptAnalyzer:
     return analyzer
 
 
+def _find_list_from_str(string: str) -> Optional[list[str]]:
+    """Finds a list from a string."""
+    start_bracket_index = string.find("[")
+    end_bracket_index = string.find("]")
+    if (
+        start_bracket_index != -1
+        and end_bracket_index != -1
+        and end_bracket_index > start_bracket_index
+    ):
+        new_list = string[start_bracket_index : end_bracket_index + 1]
+        return json.loads(new_list.replace("'", '"'))
+    return None
+
+
+def _update_tag_decorator_with_version(
+    decorators: list[str], variables: MirascopeCliVariables
+) -> Optional[str]:
+    """Updates the tag decorator and returns the import name."""
+    if variables.revision_id is None:
+        return None
+    import_name = "tags"
+    tag_exists = False
+    version_tag_prefix = "version:"  # mirascope tag prefix
```

Comment:
I feel like it makes more sense to put the colon below? so that tag prefix is `version` and the tag is f"{prefix}:{id}"

Reply:
hmmm, but if for whatever reason somebody had `version1` as their tag, it gets caught, whereas i'm reserving the `version:` keyword instead